packages/backend-core/src/db/Replication.ts

```diff
@@ -63,11 +63,15 @@ class Replication {
 
     return {
       ...opts,
-      filter: (doc: any, params: any) => {
+      filter: (doc: Document, params: any) => {
         // don't sync design documents
         if (toDev && doc._id?.startsWith("_design")) {
           return false
         }
+        // always replicate deleted documents
```

Comment:
Why this? Will this create documents that will be "immediately" deleted?

Reply:
No - it is necessary to actually delete documents in production - I ran into this issue with deleting an automation in development, then publishing and the automation wouldn't be removed.